lib/trycourier/brands.rb

```diff
@@ -0,0 +1,105 @@
+require "trycourier/version"
+require "net/http"
+require "json"
+require "openssl"
+
+module Courier
+  # generic error to catch all exceptions
+  class CourierAPIException < StandardError; end
```

Comment:
I think we only need to do this once inside the Courier module. Feels like we are redefining it unnecessarily.